devel/jenkins-cli/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           golang 1.0
 
-go.setup            github.com/jenkins-zh/jenkins-cli 0.0.44 v
+go.setup            github.com/jenkins-zh/jenkins-cli 0.0.46 v
 set git_commit      4a589b1
```

Comment:
It would be surprising if this did not change

Reply:
Sure, fixed.